livekit-plugins/livekit-plugins-deepgram/livekit/plugins/deepgram/tts.py

```diff
@@ -0,0 +1,341 @@
+from __future__ import annotations
+
+import asyncio
+import json
+import os
+from dataclasses import dataclass
+
+import aiohttp
+from livekit import rtc
+from livekit.agents import (
+    DEFAULT_API_CONNECT_OPTIONS,
+    APIConnectionError,
+    APIConnectOptions,
+    APIStatusError,
+    APITimeoutError,
+    tokenize,
+    tts,
+    utils,
+)
+
+from ._utils import _to_deepgram_url
+from .log import logger
+
+BASE_URL = "https://api.deepgram.com/v1/speak"
+NUM_CHANNELS = 1
+
+
+@dataclass
+class _TTSOptions:
+    model: str
+    encoding: str
+    sample_rate: int
+    container: str
+    word_tokenizer: tokenize.WordTokenizer
+
+
+class TTS(tts.TTS):
+    def __init__(
+        self,
+        *,
+        model: str = "aura-asteria-en",
+        encoding: str = "linear16",
+        sample_rate: int = 24000,
+        container: str = "none",
```

Comment:
will not be used since rtc.AudioFrame doesn't support it 
```suggestion
```

Reply:
![image](https://github.com/user-attachments/assets/f05247e2-ff10-4cff-9c56-c7437439e6da)
https://developers.deepgram.com/docs/tts-media-output-settings
I think it won't affect rtc.AudioFrame because it's just a way to get specific Content-Type